cmd/boots/main.go

```diff
@@ -286,6 +288,7 @@ func newCLI(cfg *config, fs *flag.FlagSet) *ffcli.Command {
 	fs.StringVar(&cfg.logLevel, "log-level", "info", "log level.")
 	fs.StringVar(&cfg.dhcpAddr, "dhcp-addr", conf.BOOTPBind, "IP and port to listen on for DHCP.")
 	fs.StringVar(&cfg.syslogAddr, "syslog-addr", conf.SyslogBind, "IP and port to listen on for syslog messages.")
+	fs.StringVar(&cfg.kernelArgs, "kernel-args", "", "Kernel args to add to iPXE initrd for OSIE.")
```

Comment:
question: Does a string make sense here or a map of string to string, or a slice of strings?

Reply:
Unfortunately, i don't believe `*flag.FlagSet` has either of those options.

Reply:
You can drop in `flag "github.com/spf13/pflag"` as a replacement, and it has both those types

Reply:
I tried to use `github.com/spf13/pflag` but the use of `github.com/peterbourgon/ff/ffcli` requires the stdlib flag package. see [here](https://github.com/peterbourgon/ff/blob/7c05c99406b871976b139e55f9a771f22769f9de/ffcli/command.go#L55).